cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperator.java

```diff
@@ -678,4 +678,16 @@ protected Future<Void> removeAnnotation(Reconciliation reconciliation, KafkaMirr
         return resourceOperator.patchAsync(reconciliation, patchedKafkaMirrorMaker2)
             .compose(ignored -> Future.succeededFuture());
     }
+
+    /**
+     * Deletes the ClusterRoleBinding which as a cluster-scoped resource cannot be deleted by the ownerReference
+     * <p>
```

Comment:
do we need this starting paragraph in HTML style?

Reply:
I don't think so. This was copied from the Connect method where I deleted it later. I removed it here as well.